### PR TITLE
A stats dialog to keep track of your record

### PR DESCRIPTION
**Pull request by jnsturgis**
Hi I have tried to add a stats dialog to show how well you are performing along the lines of the real Wordle one.. I hope you enjoy and above all any ideas to improve the functionality - or coding style (I have not tried javascript/typescript or tailwind and React before so it is a learning experience...

### CODE REVIEW

File: src/components/histogram/histogram.tsx [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+
+type Props = {
+  data: number[]
+}
+
+export const Histogram = ({ data }: Props) => {
+  const min = 10
+  const max = Math.ceil(Math.max.apply(null, data)*1.2)
+  return(
+    <div className="columns-1 justify-left m-2 bg-blue-300 text-sm">
```
Comment (hannahcode) [inline src/components/histogram/histogram.tsx:11]: Style edit - remove background color here:
```suggestion
    <div className="columns-1 justify-left m-2 text-sm">
```

File: src/components/histogram/progress.tsx [hunk 2]
```diff
@@ -0,0 +1,23 @@
+
+type Props = {
+  index: number,
+  size: number,
+  label: string
+}
+
+export const Progress = ( {index, size, label}: Props ) => {
+  return(
+    <div className="flex justify-left m-1 bg-blue-200">
```
Comment (hannahcode) [inline src/components/histogram/progress.tsx:10]: Style edit - remove background color here:
```suggestion
    <div className="flex justify-left m-1">
```

**Comment by jnsturgis** [comment]
Yes I got a bit carried away with shades of blue trying to get things properly positioned

File: src/components/statline/statline.tsx [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+type Props = {
+  labels: string[]
+  values: string[]
+}
+
+export const StatLine = ({labels, values}: Props) => {
+  return (
+    <div className="flex justify-center m-1">
+    {values.map((value,i ) => (
+       <div key={i} className="items-center justify-center m-1 w-1/4 bg-blue-300">
```
Comment (hannahcode) [inline src/components/statline/statline.tsx:12]: One more background removal - 

```suggestion
       <div key={i} className="items-center justify-center m-1 w-1/4">
```

**Comment by hannahcode** [comment]
Thanks again!

**Pull request merged**